packages/web/src/components/remix-settings-modal/RemixSettingsModal.tsx

```diff
@@ -127,7 +127,8 @@ const RemixSettingsModal = ({
 
```

Comment:
I didnt change anything in here but eslint flagged it 😠
so I just formatted it  🤷